.github/workflows/integration-tests.yml

```diff
@@ -825,21 +826,24 @@ jobs:
 
       - name: Prepare
         run: |
-          rustup toolchain install nightly
+          # The last nightly rust that still support "-Zprofile"
+          #
+          # See https://github.com/rust-lang/rust/pull/131829
+          rustup toolchain install nightly-2024-11-01
           cargo new coverage-test
           cd coverage-test
           echo "serde = { version = \"1.0\", features = [\"derive\"] }" >> Cargo.toml
 
       - name: "Coverage test #1"
         working-directory: ./coverage-test
-        run: cargo clean && cargo +nightly test
+        run: cargo clean && cargo +nightly-2024-11-01 test
 
       - name: Output
         run: ${SCCACHE_PATH} --show-stats
 
       - name: "Coverage test #2"
         working-directory: ./coverage-test
-        run: cargo clean && cargo +nightly test
+        run: cargo clean && cargo +nightly-2024-11-01 test
```

Comment:
maybe document why Nov 1st?

Reply:
Added before. Let's me extra it as an env.